versions/1/data_stream/manifest.spec.yml

```diff
@@ -111,6 +111,10 @@ spec:
       type: string
       examples:
       - diagnostics
+    dataset_is_prefix:
+      description: if true, the generated index pattern will contain the dataset as a prefix only
```

Comment:
Is this only about the field `index_patterns` in an ES index template (this is what I guessed from the discussions), or also about Kibana index patterns (this is how this line reads without context)?

Reply:
It is only about the ES index patterns. It should not affect the Kibana ones as these are based on the types and include all datasets anyways.

Reply:
Then this description should be changed accordingly.